Test-04.09/Task3/Task3.c

```diff
@@ -0,0 +1,36 @@
+﻿#include <stdio.h>
+#include <stdbool.h>
+
+int main() {
+    FILE* file;
```

Comment:
```suggestion
    FILE* file = NULL;
```
А то ненициализированная переменная приносит минус балл сразу :)